src/metabase/lib/js/metadata.cljs

```diff
@@ -215,7 +215,7 @@
 (defmethod excluded-keys :card
   [_object-type]
   #{:database
-    :dimension_options
+    :dimension-options
```

Comment:
I guess this was just a typo.

Reply:
I think it was something I missed changing after the PR that switched everything to `kebab-case`/`camelCase`